rest/src/routes/finalizationRoutes.js

```diff
@@ -35,18 +38,31 @@ module.exports = {
 		const { connections } = services;
 		const { timeout } = services.config.apiNode;
 
+		const readAndParseNetworkPropertiesFile = () => {
+			const readFile = util.promisify(fs.readFile);
+			try {
```

Comment:
Is it possible to move the read of configuration files to a service? 

1) be reused here and in the network routes
2) having the object cached in memory, for some reason the file is being read every request.

Reply:
thought about it, but this is a simple temporal patch for testnet/v3 only. won't probably need it once mainnet launched.

Reply:
Ok, add a comment "to be removed after main merge" or similar.